ads/bannerflow.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  writeScript,
+  validateSrcPrefix,
+} from '../3p/3p';
+
+
+export function bannerflow(global, data) {
+  const src = data.src;
```

Comment:
please validate data.src before calling `#validateSrcPrefix()`. You can use the `#validateData()` function